x/ecocredit/server/marketplace/utils.go

```diff
@@ -120,16 +122,6 @@ func (k Keeper) fillOrder(ctx context.Context, sellOrder *api.SellOrder, buyerAc
 			return err
 		}
 		buyerBal.Tradable = tradableBalance.String()
-
-		supplyTradable, err := math.NewDecFromString(supply.TradableAmount)
-		if err != nil {
-			return err
-		}
-		supplyTradable, err = math.SafeAddBalance(supplyTradable, purchaseQty)
-		if err != nil {
-			return err
-		}
-		supply.TradableAmount = supplyTradable.String()
```

Comment:
heres the bug, tradable is the same as escrowed so there shouldn't have been an edit to supply in this case

Reply:
Thanks for pointing out and adding a comment.